services/clusters-keeper/src/simcore_service_clusters_keeper/models.py

```diff
@@ -13,17 +19,60 @@ class EC2InstanceType:
 
 
 InstancePrivateDNSName = str
+EC2Tags: TypeAlias = dict[str, str]
 
 
 @dataclass(frozen=True)
 class EC2InstanceData:
     launch_time: datetime.datetime
     id: str  # noqa: A003
     aws_private_dns: InstancePrivateDNSName
+    aws_public_ip: str | None
     type: InstanceTypeType  # noqa: A003
     state: InstanceStateNameType
+    tags: EC2Tags
 
 
-@dataclass(frozen=True)
-class Cluster:
-    ...
+class ClusterState(StrAutoEnum):
+    STARTED = auto()
+    RUNNING = auto()
+    STOPPED = auto()
+
+
+def _convert_ec2_state_to_cluster_state(
+    ec2_state: InstanceStateNameType,
+) -> ClusterState:
+    match ec2_state:
+        case "pending":
+            return ClusterState.STARTED  # type: ignore
+        case "running":
+            return ClusterState.RUNNING  # type: ignore
+        case _:
+            return ClusterState.STOPPED  # type: ignore
+
+
+class ClusterGet(BaseModel):
+    endpoint: AnyUrl
```

Comment:
MINOR: we would like to reach some kind of conventions with url fields in the API shemas.

These are the proposals for suffixes so far
`*_url`: url field
`*_id` (or `idr` if full-name): identifier field
`*_at` : datetime field

Reply:
I actually really don't understand _url, or _id, when there is already a type giving this away.
the _at I could live with it.